nusacrowd/nusa_datasets/indsp_news_ethnicsr/indsp_news_ethnicsr.py

```diff
@@ -0,0 +1,206 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import json
+import os
+
+from nusacrowd.utils import schemas
+from nusacrowd.utils.configs import NusantaraConfig
+from nusacrowd.utils.constants import Tasks
+from zipfile import ZipFile
+
+_CITATION = """\
+@inproceedings{sani-cocosda-2012,
+    title = "Towards Language Preservation: Preliminary Collection and Vowel Analysis of {I}ndonesian Ethnic Speech Data",
+    author = "Sani, Auliya and Sakti, Sakriani and Neubig, Graham and Toda, Tomoki and Mulyanto, Adi and Nakamura, Satoshi",
+    booktitle = "Proc. Oriental COCOSDA",
+    year = "2012",
+    pages = "118--122"
+    address = "Macau, China"
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ind"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+_DATASETNAME = "indsp_news_ethnicsr"
+
+_DESCRIPTION = """
+INDspeech_NEWS_EthnicSR is a collection of Indonesian ethnic speech corpora for Javanese and Sundanese for Indonesian ethnic speech recognition. It was developed in 2012 by the Nara Institute of Science and Technology (NAIST, Japan) in collaboration with the Bandung Institute of Technology (ITB, Indonesia) [Sani et al., 2012].
+"""
+
+_HOMEPAGE = "https://github.com/s-sakti/data_indsp_news_ethnicsr"
+
+_LICENSE = "CC-BY-NC-SA 4.0"
+
+_URLS = {
+    _DATASETNAME: "https://github.com/s-sakti/data_indsp_news_ethnicsr/archive/refs/heads/main.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.SPEECH_RECOGNITION]
+_SOURCE_VERSION = "1.0.0"
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class indsp_news_ethnicsr(datasets.GeneratorBasedBuilder):
+    """INDspeech_NEWS_EthnicSR is a collection of Indonesian ethnic speech corpora for Javanese and Sundanese for Indonesian ethnic speech recognition. It was developed in 2012 by the Nara Institute of Science and Technology (NAIST, Japan) in collaboration with the Bandung Institute of Technology (ITB, Indonesia) [Sani et al., 2012]."""
+
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    NUSANTARA_VERSION = datasets.Version(_NUSANTARA_VERSION)
+    
+    BUILDER_CONFIGS = []
+    TYPE_LIST = ['Jawa', "Sunda"]
```

Comment:
Could you change `Jawa` into `jv` and `Sunda` into `su` following the naming convention of other multilingual datasets??